doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_custom_pcb.rst

```diff
@@ -75,17 +76,325 @@ This is useful when making a change on the PCB (for example, when changing the c
++=================================+
+| Integer [pF], min 1, max 25     |
++---------------------------------+
+
+.. note::
+   In the JSON configuration, the load capacitance parameter is only applied if the option ``builtInLoadCapacitors`` is explicitly set to true.
+
+TIME (LFXO startup time)
+^^^^^^^^^^^^^^^^^^^^^^^^
+
++-----------------------------+
+| JSON (startupTimeMs)        |
++=============================+
+| Integer [ms], min 1, max 25 |
++-----------------------------+
+
+See the following example:
+
+.. code-block:: json
+
+   {
+     "lfosc": {
+       "source": "LFXO",
+       "lfxo": {
+         "mode": "CRYSTAL",
+         "accuracyPPM": 20,
+         "startupTimeMs": 600,
+         "builtInLoadCapacitancePf": 15,
+         "builtInLoadCapacitors": true
+       }
+     }
+   }
+
+Source
+^^^^^^
+
+The ``source`` option can be one of the following:
+
+* ``LFXO``, when an external crystal oscillator is in place.
+* ``LFRC``, when an external Crystal Oscillator is not in place.
+
+This means that the device can use either ``LFRC`` or ``SYNTH`` as clock sources.
+
+LFRC autocalibration configuration
+----------------------------------
+
+The three values provided correspond to the LFRC autocalibration configuration:
+
+* ``temp-interval`` specifies the interval between temperature measurements, expressed in 0.25-second increments.
+* ``temp-delta`` defines the temperature change, in 0.25-degree Celsius steps, that triggers a calibration event.
+* ``interval-max-count`` indicates the maximum number of measurement intervals allowed between calibrations, regardless of temperature variations.
+
+Use these parameters to precisely control the LFRC autocalibration behavior.
+
+Datasheet register field
+^^^^^^^^^^^^^^^^^^^^^^^^
+
++---------------------+------------------------------------+-------------------------------+
+| Datasheet field     | JSON variable                      | JSON value                    |
++=====================+====================================+===============================+
+| TEMPINTERVAL (A)    | tempMeasIntervalSeconds            | 5                             |
++---------------------+------------------------------------+-------------------------------+
+| TEMPDELTA (B)       | tempDeltaCalibrationTriggerCelsius | 10                            |
++---------------------+------------------------------------+-------------------------------+
+| INTERVALMAXNO (C)   | maxMeasIntervalBetweenCalibrations | 3                             |
++---------------------+------------------------------------+-------------------------------+
+| ENABLE              | calibrationEnabled                 | -                             |
++---------------------+------------------------------------+-------------------------------+
+
+LFRC Autocalibration is not configured in the JSON configuration files for the DK, as the default values (``4``, ``0.5`` and ``2``) will be good enough for most use-cases.
```

Comment:
Check whether it should be "LFRC Autocalibration" like there, or "LFRC autocalibration" used earlier, and make sure it is capitalized consistently.